docs/reference/plugins.md

```diff
@@ -22,22 +22,48 @@ Siehe [MBMD](https://github.com/volkszaehler/mbmd) für ein Beispiel wie man Mod
 **Beispiel Lesen**:
 
 ```yaml
-source: mqtt
-topic: mbmd/sdm1-1/Power
-timeout: 30s # don't accept values older than timeout
-scale: 0.001 # floating point factor applied to result, e.g. for Wh to kWh conversion
+meters:
+- name: grid-meter
+  type: custom
+  power:
+    source: mqtt
+    topic: mbmd/sdm1-1/Power # MQTT topic auf dem der Wert empfangen wird
+    timeout: 30s # wie alt empfangene Wert maximal 
```

Comment:
Das gehört nicht in ein Beispiel sondern in eine grundsätzliche Erläuterung.

Reply:
Gerne, genauso wie beim HTTP plugin nehme ich an.